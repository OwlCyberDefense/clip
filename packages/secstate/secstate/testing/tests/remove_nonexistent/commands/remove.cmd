/usr/sbin/secstate remove fjdskafdjskl
