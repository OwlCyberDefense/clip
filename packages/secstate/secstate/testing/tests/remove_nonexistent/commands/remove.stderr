Could not find fjdskafdjskl in database
